zerver/views/invite.py

```diff
@@ -232,6 +233,55 @@ def resend_user_invite_email(
     return json_success(request)
 
 
+@require_member_or_admin
+@typed_endpoint
+def edit_multiuse_invite(
+    request: HttpRequest,
+    user_profile: UserProfile,
+    *,
+    invite_id: PathOnly[int],
+    stream_ids: Json[list[int]] | None = None,
+    invite_as: Annotated[
+        Json[int] | None,
+        check_int_in_validator(list(PreregistrationUser.INVITE_AS.values())),
+    ] = None,
+    include_realm_default_subscriptions: Json[bool] = False,
```

Comment:
I am not sure about the default here while editing. If user doesn't pass this value thinking they do not want to change it, it would automatically be set to `False`. I think this should have default of `None` as well like other values.